include/mfmg/dealii/anasazi.templates.hpp

```diff
@@ -78,6 +79,12 @@ AnasaziSolver<OperatorType, VectorType>::solve(
   Anasazi::Eigensolution<double, MultiVectorType> solution =
       solver->getProblem().getSolution();
 
+  int const verbosity = params.get("verbosity", 0);
```

Comment:
Again, I think it's useful to see this option in `hierarchy_input.info`.

Reply:
Sure I can add these two in `hierarchy_input.info` file.